orc8r/cloud/go/tools/combine_swagger/spec/combine.go

```diff
@@ -17,187 +17,56 @@ import (
 	"io/ioutil"
 	"os"
 	"path/filepath"
-	"regexp"
-	"sort"
 	"strings"
-
-	"magma/orc8r/cloud/go/tools/swaggergen/generate"
-
-	"github.com/hashicorp/go-multierror"
-	"github.com/pkg/errors"
-	"gopkg.in/yaml.v2"
 )
 
-// Load specs from file to Swagger structs.
-func Load(inDir, commonFilepath string) ([]generate.SwaggerSpec, generate.SwaggerSpec, error) {
-	specs, err := loadSpecsFromInputDir(inDir)
+// Load a common swagger spec file and a directory of
```

Comment:
Since tools/combine_swagger/spec pkg no longer actually does much in the way of defining or operating on swagger specs, we can rename the pkg (directory and pkg name) to e.g. `combine` -- so now it'd be tools/combine_swagger/combine